### PR TITLE
[FEATURE] make news detail page navigation display next and prev news…

### PR DESCRIPTION
**Pull request by aennafae**
… item title & date

### CODE REVIEW

File: Resources/Private/Extensions/News/Templates/News/Detail.html [hunk 1]
```diff
@@ -189,40 +189,55 @@ <h2>
...
+
+			<f:if condition="{settings.backPid}">
+				<!-- Link Back -->
+				<div class="news-backlink-wrap">
+					<f:link.page pageUid="{settings.backPid}">
+						<f:translate key="back-link" />
+					</f:link.page>
+				</div>
+			</f:if>
+		</f:then>
```
Comment (dmh) [inline Resources/Private/Extensions/News/Templates/News/Detail.html:240]: Looks like this tag `</f:then>` is not needed - cause errors.

**Review by dmh — CHANGES_REQUESTED** [review]

File: Resources/Private/Extensions/News/Templates/News/Detail.html (hunk 1 above)
Comment (dmh) [reply Resources/Private/Extensions/News/Templates/News/Detail.html:240]: Also, could you explain how to enable navigation for next and prev news in detail view template? I would like to test it, but can't find this feature :)

Comment (dkd-oertel) [reply Resources/Private/Extensions/News/Templates/News/Detail.html:240]: Do you mean "_plugin.tx_news.settings.detail.showPrevNext = 1_" in EXT:theme_t3kit/Resources/Private/Extensions/News/TypoScript/setup.ts? It should be enabled.

Comment (dkd-oertel) [reply Resources/Private/Extensions/News/Templates/News/Detail.html:240]: Do you still have problems to test it ?

Comment (dmh) [reply Resources/Private/Extensions/News/Templates/News/Detail.html:240]: Yes, I still can't enable those links, looks like I am missing some config ... :(
But I decided to merge it without testing, hope that everything should be ok. Thanks for the help!

**Review by dmh — APPROVED** [review]

**Pull request merged**